astropy/table/column.py

```diff
@@ -1108,7 +1108,7 @@ def _represent_as_dict(self):
                 out['mask'] = col.mask
                 self._represent_as_dict_attrs += ('mask',)
 
-        elif method is 'null_value':
+        elif method == 'null_value':
```

Comment:
Nice catch. Coincidentally, also caught by @taldcroft in a PR by me: https://github.com/astropy/astropy/pull/8998#discussion_r324299855